charts/agent/tests/global_overrides_test.yaml

```diff
@@ -0,0 +1,59 @@
+suite: Test global value overrides
```

Comment:
nit: this is a little nebulous. maybe change to something like `Test subchart overrides of global values`? this nit applies for all suite names.